snap/snapcraft.yaml

```diff
@@ -0,0 +1,134 @@
+name: edgex-device-usb-camera
+base: core20
+license: Apache-2.0
+adopt-info: metadata
```

Comment:
Do the `summary` and `description` also come from the `metadata` part?

Reply:
yes